worker/src/main/java/gov/cms/ab2d/worker/processor/JobProcessorImpl.java

```diff
@@ -108,19 +107,21 @@ public Job process(final String jobUuid) {
     /**
      * Process in individual contract
      *
-     * @param contract - the contract to process
-     * @param job - the job in which the contract belongs
-     * @param month - the month to search for beneficiaries for
-     * @param outputDirPath - the location of the job output
+     * @param job             - the job in which the contract belongs
+     * @param month           - the month to search for beneficiaries for
+     * @param outputDirPath   - the location of the job output
      * @param progressTracker - the progress tracker which indicates how far the job is along
-     * @throws ExecutionException when there is an issue with searching
+     * @throws ExecutionException   when there is an issue with searching
      * @throws InterruptedException - when the search is interrupted
      */
-    void processContract(Contract contract, Job job, int month, Path outputDirPath, ProgressTracker progressTracker) throws ExecutionException, InterruptedException {
+    void processContract(Job job, int month, Path outputDirPath, ProgressTracker progressTracker)
+            throws ExecutionException, InterruptedException {
+        Contract contract = job.getContract();
+        assert contract != null;
```

Comment:
Missed the assert

Reply:
Asserts are intentional.  With the new assumption that a User is bound to a single Contract, results in a Job being bound to a single contract.

The contract reference must exist.